src/solver.js

```diff
@@ -1154,8 +1163,13 @@ Tableau.prototype.setEvaluation = function () {
     // Rounding objective value
     var roundingCoeff = Math.round(1 / this.precision);
     var evaluation = this.matrix[this.costRowIndex][this.rhsColumn];
-    this.evaluation =
+    var roundedEvaluation =
         Math.round(evaluation * roundingCoeff) / roundingCoeff;
+
+    this.evaluation = roundedEvaluation;
+    if (this.simplexIters === 0) {
+        this.relaxedSolution = roundedEvaluation;
```

Comment:
For performance reasons, the ```relaxedSolution``` property should be initialised in the constructor of the ```Tableau```. may be:
```javascript
this.relaxedSolution = 0;
```

Also the name ```relaxedSolution``` is not completely accurate. We call "solution" the set of values for the decision variables. Here I would call the variable ```relaxedEvaluation```. We call "evaluation" the value of the objective function for a given solution.